src/api/controllers/authController.js

```diff
@@ -0,0 +1,65 @@
+import bcrypt, { hashSync, genSaltSync } from 'bcrypt';
+import models from '../models/index';
+import generateToken from '../../helpers/tokens/generate.token';
+import sendResult from '../../helpers/results/send.auth';
+
+const { User } = models;
+
+/**
```

Comment:
Can you please add a description of the functions in your ```JsDocs```?

Reply:
done